src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,114 @@
+import cn from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+import { FC } from 'react';
+import { PersonLink } from '../PersonLink';
+import { SearchLink } from '../SearchLink';
+import { Person } from '../../types';
+import { SearchFilterParams } from '../../types/SearchFilterParams';
+
+interface Props {
+  people: Person[];
+}
+
+export const PeopleTable: FC<Props> = ({ people }) => {
+  const [searchParams] = useSearchParams();
+  const sort = searchParams.get(SearchFilterParams.Sort)
+  || SearchFilterParams.None;
+  const order = searchParams.get(SearchFilterParams.Order)
+  || SearchFilterParams.None;
+
+  const getLinkClass = (param: string) => cn(
+    'fas',
+    { 'fa-sort': param !== sort },
+    { 'fa-sort-up': sort && sort === param && !order },
+    { 'fa-sort-down': sort && sort === param && order === 'desc' },
+  );
+
+  const getLinkParams = (
+    param: string,
+  ):
+  | { sort: string }
+  | {
+    sort: null;
+    order: null;
+  }
+  | { order: string } => {
+    if (!sort) {
+      return { sort: param };
+    }
+
+    if (sort === param) {
+      return order ? { sort: null, order: null } : { order: 'desc' };
+    }
+
+    return { sort: param };
```

Comment:
```suggestion
    if (sort === param) {
      return order ? { sort: null, order: null } : { order: 'desc' };
    }

    return { sort: param };
```
You return `{ sort: param }` if all cases if `sort !== param`